src/routes/acquirerRoutes.ts

```diff
@@ -53,8 +54,20 @@ const skipThisStep = (step: string, formdata: FormData) => {
       const legalGatewayStep = formdata.steps["legal-gateway"].value as LegalGatewayStep
       return legalGatewayStep.yes.checked || legalGatewayStep.other.checked
     }
+    case "lawful-basis-personal": {
+      // Skip lawful-basis-personal if answer to data-type was "special" or "none"
+      const DataTypeStep = formdata.steps["data-type"].value as DataTypeStep
+      return DataTypeStep.special.checked || DataTypeStep.none.checked
+    }
+    case "lawful-basis-special": {
+      // Skip lawful-basis-special if answer to data-type was "personal" or "none"
+      const DataTypeStep = formdata.steps["data-type"].value as DataTypeStep
+      return DataTypeStep.personal.checked || DataTypeStep.none.checked
```

Comment:
same happens here, what will skip is special to be false, if you select personal and special  lawful-basis-special will also work. And if we are not allowed to select none and special at same time you could just write: DataTypeStep.special.checked === false